source/reference/minio-mc-admin/mc-admin-replicate.rst

```diff
@@ -80,6 +80,8 @@ Syntax
    :fullpath:
 
    Create or expand a site replication configuration.
+   New configurations use asynchronous site replication.
+   To enable synchronous replication for an existing configuration, see :mc-cmd:`~mc admin replicate update --sync`.
```

Comment:
I'm meh about this wording for these two sentences. Suggestions appreciated.

Reply:
> New site replication configurations use asynchronous mode by default, as MinIO recommends.
> If your circumstances require synchronous replication, complete the configuration, then use :mc-cmd:`mc admin replicate update --sync`.

Reply:
Went with a mashup of the two, as `add` doesn't have `--sync`. Only `update` does.

Reply:
Completely slipped past me. That's an interesting design choice.